New PS cmdlet and updates to existing (#9243)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Import-CsOnlineAudioFile.md

## PARAMETERS

### -ApplicationId

> The ApplicationId parameter is the identifier for the application which will use this audio file. For example, if the audio file will be used with an Auto Attendant, then it needs to be set to "OrgAutoAttendant". If the audio file will be used with a Call Queue, then it needs to be set to "HuntGroup". If the audio file will be used with Microsoft Teams, then it needs to be set to "TenantGlobal"

Reviewer: ```suggestion
The ApplicationId parameter is the identifier for the application which will use this audio file. For example, if the audio file will be used with an Auto Attendant, then it needs to be set to "OrgAutoAttendant". If the audio file will be used with a Call Queue, then it needs to be set to "HuntGroup". If the audio file will be used with Microsoft Teams, then it needs to be set to "TenantGlobal."
```

### Microsoft.Rtc.Management.Hosted.Online.Models.AudioFile

## NOTES

> When you are importing an audio file to be used for Auto Attenant or Call Queue, the audio file will automatically be marked for deletion (as seen by running

Reviewer 2: ```suggestion
When you are importing an audio file to be used for Auto Attendant or Call Queue, the audio file will automatically be marked for deletion (as seen by running
```

> When you are importing an audio file to be used for Auto Attendant or Call Queue, the audio file will automatically be marked for deletion (as seen by running

Reviewer: ```suggestion
When you import an audio file to be used for Auto Attendant or Call Queue, the audio file will automatically be marked for deletion (as seen by running
```
